aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -166,6 +166,15 @@ Tensor empty_like(const Tensor& self, const TensorOptions& options) {
   return at::empty(self.sizes(), options);
 }
 
+Tensor empty_like(const Tensor& self, const TensorOptions& options, MemoryFormat memory_format) {
```

Comment:
The defaulting behavior for `_like` operations is non-obvious and I wish it were prominently documented somewhere.